src/LodeRunner.Data/Interfaces/ITestRunService.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using LodeRunner.Core;
+using LodeRunner.Core.Models;
+using LodeRunner.Core.Responses;
+
+namespace LodeRunner.Data.Interfaces
+{
+    /// <summary>
+    /// TestRun Interface.
+    /// </summary>
+    public interface ITestRunService
```

Comment:
It looks to me like we don't need this interface or `ILoadTestConfigService` or `IClientStatusService`.  The Post, Terminate, and Delete methods could all be part of `IBaseService`.  For services that don't implement a specific member of the interface we can throw a not implemented error for the implementation.

Also, even if we keep the different service specific interfaces there is inconsistency in naming.  For example, `Post()` and `PostUpdate()` both exist, but seem to be the same method declaration.  I think it would be better to just move them to IBaseService as generics and get rid of the service-specific interfaces that have been added.

Reply:
@jofultz will be addressing this refactor in this PR.